python/conv2d.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+import warnings
+warnings.filterwarnings("ignore")
+import tensorflow as tf
+from tensorflow import keras
+tf.compat.v1.enable_eager_execution()
+import matplotlib.pyplot as plt
+from model_utils import save_model
+
+n_frames = 50
+n_features = 23
+
+# construct TensorFlow model
+model = keras.Sequential()
+model.add(keras.layers.InputLayer(input_shape=(n_frames, n_features, 1)))
+model.add(keras.layers.Conv2D(2, (5, 5), dilation_rate=(2, 1), strides=(1, 1), padding='valid', kernel_initializer='random_normal', bias_initializer='random_normal', activation='relu'))
+model.add(keras.layers.Conv2D(3, (4, 3), dilation_rate=(1, 1), strides=(1, 2), padding='same', kernel_initializer='random_normal', bias_initializer='random_normal'))
+model.add(keras.layers.BatchNormalization(momentum=0.0, epsilon=0.01, beta_initializer='random_normal', gamma_initializer='glorot_uniform', moving_mean_initializer="random_normal", moving_variance_initializer="ones"))
```

Comment:
Would it be possible to add another 2D BatchNorm layer, just to test the "affine" code path?

Reply:
I think it's the other way around. Here the `affine == true` is tested and the non affine path needs to be tested.

In `python=`conv.py`, the first BN is affine and the second is not (`center=False, scale=False`

Reply:
Done